http://sourceware.org/ml/gdb-cvs/2012-03/msg00296.html

### src/gdb/doc/ChangeLog	2012/03/22 08:10:41	1.1289
### src/gdb/doc/ChangeLog	2012/03/27 20:15:20	1.1290
## -1,3 +1,12 @@
+2012-03-27  Jan Kratochvil  <jan.kratochvil@redhat.com>
+
+	* gdb.texinfo (Auto-loading): Rename node reference
+	'.debug_gdb_scripts section' to 'dotdebug_gdb_scripts section'.
+	Twice.
+	(.debug_gdb_scripts section): Rename the node ...
+	(dotdebug_gdb_scripts section): ... here.
+	(Maintenance Commands): Also rename this node reference.
+
 2012-03-22  Siva Chandra Reddy  <sivachandra@google.com>
 
 	* gdb.texinfo (Python API/Values From Inferior): Add description
--- src/gdb/doc/gdb.texinfo	2012/03/22 08:10:41	1.936
+++ src/gdb/doc/gdb.texinfo	2012/03/27 20:15:20	1.937
@@ -24717,8 +24717,8 @@
 @file{@var{objfile}-gdb.py} and @code{.debug_gdb_scripts} section.
 
 @menu
-* objfile-gdb.py file::         The @file{@var{objfile}-gdb.py} file
-* .debug_gdb_scripts section::  The @code{.debug_gdb_scripts} section
+* objfile-gdb.py file::          The @file{@var{objfile}-gdb.py} file
+* dotdebug_gdb_scripts section:: The @code{.debug_gdb_scripts} section
 * Which flavor to choose?::
 @end menu
 
@@ -24744,7 +24744,7 @@
 
 Also printed is the list of scripts that were mentioned in
 the @code{.debug_gdb_scripts} section and were not found
-(@pxref{.debug_gdb_scripts section}).
+(@pxref{dotdebug_gdb_scripts section}).
 This is useful because their names are not printed when @value{GDBN}
 tries to load them and fails.  There may be many of them, and printing
 an error message for each one is problematic.
@@ -24795,7 +24795,7 @@
 So your @file{-gdb.py} file should be careful to avoid errors if it
 is evaluated more than once.
 
-@node .debug_gdb_scripts section
+@node dotdebug_gdb_scripts section
 @subsubsection The @code{.debug_gdb_scripts} section
 @cindex @code{.debug_gdb_scripts} section
 
@@ -33475,7 +33475,7 @@
 matching @var{regexp}.
 For each script, this command prints its name as specified in the objfile,
 and the full path if known.
-@xref{.debug_gdb_scripts section}.
+@xref{dotdebug_gdb_scripts section}.
 
 @kindex maint print statistics
 @cindex bcache statistics
